tests/integration/modules/test_pkg.py

```diff
@@ -109,11 +109,11 @@ def test_mod_del_repo(self, grains):
                     grains["osmajorrelease"]
                 )
                 baseurl = (
-                    "http://repo.saltstack.com/yum/redhat/{}/x86_64/latest/".format(
+                    "http://repo.saltproject.io/py3/redhat/{}/x86_64/latest/".format(
                         grains["osmajorrelease"]
                     )
                 )
-                gpgkey = "https://repo.saltstack.com/yum/rhel{}/SALTSTACK-GPG-KEY.pub".format(
+                gpgkey = "https://repo.saltproject.io/py3/rhel{}/SALTSTACK-GPG-KEY.pub".format(
```

Comment:
I don't believe this URL exists, I think it needs to be:

`"https://repo.saltproject.io/py3/redhat/{}/x86_64/latest/SALTSTACK-GPG-KEY.pub".format(` ?

Reply:
it doesn't exist, but it didn't before.  I can change it to an existing url though.

not sure why the tests pass now with a non-existent url.